tmol/score/dunbrack/potentials/compiled.hh

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
```

Comment:
this is duplicated from dispatch.hh?

Reply:
Looks like. I axed it just now along w/ compiled.pybind.cpp